VisionOnEdge/ui/src/components/ImageLink.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { FC } from 'react';
+import { Link } from 'react-router-dom';
+import { Flex, FlexItem, Text, Image } from '@fluentui/react-northstar';
+
+interface ImageLinkProps {
+  to: string;
+  imgSrc?: string;
+  defaultSrc?: string;
+  label?: string;
+  imgPadding?: string;
+  width: string;
+  height?: string;
+}
+const ImageLink: FC<ImageLinkProps> = ({
+  to,
+  imgSrc,
+  defaultSrc,
+  label = '',
+  imgPadding = '10px',
+  width = '100px',
+  height = '100px',
+}) => {
+  return (
+    <Flex column styles={{ width }}>
+      <Link to={to}>
+        <FlexItem styles={{ height }}>
```

Comment:
The images ratios looks wrong by using this method, I suggest the method below:
```
        <div style={{ height, width }}>
          <Image
            src={imgSrc ?? defaultSrc}
            styles={{ padding: imgPadding, width: '100%', height: '100%', objectFit: 'contain' }}
          />
        </div>
```

Reply:
It looks better now. Thanks for the advice.